frontend/app/src/premium/premium-apis.ts

```diff
@@ -89,8 +90,8 @@ export const userSettings = (): UserSettingsApi => {
     scrambleData: computed<boolean>(
       () => (store.state as any).session.scrambleData
     ),
-    darkModeEnabled: computed<boolean>(() => {
-      return store.getters['settings/darkModeEnabled'];
+    selectedTheme: computed<Theme>(() => {
+      return store.getters['settings/selectedTheme'];
```

Comment:
We need to find some time at some point to do a settings pinia/composition refactor.

Reply:
Strongly agree 😂